### ISSUE TITLE
subtest doesn't print header

### ISSUE THREAD

**Issue by Leont** [rakudo/rakudo#4266]
## The Problem

In other TAP implementations, a subtest always prints a header first (one of the TAP drafts even mandated this). E.g.
```
$ perl -MTest::More -e 'subtest "foo", sub { ok 1 }; done_testing'
# Subtest: foo
    ok 1
    1..1
ok 1 - foo
1..1
```

However, `Test.rakumod` doesn't:

```
$ raku -MTest -e 'subtest "foo", { ok 1 }; done-testing'
    ok 1 - 
    1..1
ok 1 - foo
1..1
```
I would argue that it should, not only because it's consistent but also because it's makes analysing what's going wrong much more easy when one can actually find the name of the subtest.

Note that this should be printed to `$*STDOUT`, not `$*STDERR`, so `diag` is not appropriate here.

## Environment
```
Welcome to Rakudo(tm) v2020.12.
Implementing the Raku(tm) programming language v6.d.
Built on MoarVM version 2020.12.
```

**Comment by vrurg** [comment]
Implemented by #4280.

**Comment by vrurg** [comment]
#4280 is merged, closing.
